bokehjs/src/lib/models/glyphs/image_url.ts

```diff
@@ -74,6 +75,32 @@ export class ImageURLView extends XYGlyphView {
       }
       img.src = this._url[i]
     }
+
+    const xs : Array<number> = []
```

Comment:
For consistency across code base, the type should be `number[]`. This is actually enforced by a tslint rule, though tslint isn't enabled on travis ci (this is fixed in PR #8085).

Reply:
Also, whitespace.